### PR TITLE
resolve #135

### LINKED ISSUE #135: Parametrize application.conf variables

### PR DESCRIPTION
**Pull request by carueda**
at least in terms of what's indicated in the README re overwriting of certain defaults

> You can override the defaults with the following > environment variables:
> 
> - API_HOST
> - API_PORT
> - DB_HOST
> - DB_PORT
> - DB_NAME
> - DB_USER
> - DB_PASSWORD

### CODE REVIEW

**Review by lemastero — APPROVED** [review]
LGTM

File: src/main/g8/src/main/resources/application.conf [hunk 1]
```diff
@@ -6,16 +6,24 @@ akka {
...
 db {
   db_name = "items"
+  db_name = ${?DB_NAME}
   host = "localhost"
+  host = ${?DB_HOST}
   port = 5432
-  url = "jdbc:postgresql://"\${db.host}":"\${db.port}"/"\${db.db_name}
+  port = ${?DB_PORT}
+  
+  url = "jdbc:postgresql://"${db.host}":"${db.port}"/"${db.db_name}
```
Comment (jczuchnowski) [inline src/main/g8/src/main/resources/application.conf:22]: The backslashes before `$` are needed for escaping. Without it the template fails to generate a project.
The tests should pick that up but somethings wrong with them at the moment.

**Review by jczuchnowski — CHANGES_REQUESTED** [review]
Thanks for the contribution! Just a small thing to fix.

File: src/main/g8/src/main/resources/application.conf (hunk 1 above)
Comment (carueda) [reply src/main/g8/src/main/resources/application.conf]: Thanks for the explanation (first time contributing to a g8!),
I've just escaped all those`$`.

**Review by jczuchnowski — APPROVED** [review]

**Pull request merged**